eval/mt_bench/components.py

```diff
@@ -14,16 +14,19 @@ def run_mt_bench_op(
     merge_system_user_message: bool,
     max_workers: str = "auto",
     device: str = None,
-):
-    def launch_vllm_server_background(model_path: str, retries: int = 60, delay: int = 5) -> int:
+) -> NamedTuple('outputs', best_model=str, best_score=float):
+
+    import subprocess
+    subprocess.run(["python3.11", "-m", "pip", "install", "vllm"], check=True)
+
+    # TODO: Ensure vLLM server is utilizing GPU
+    def launch_vllm_server_background(model_path: str, retries: int = 60, delay: int = 5):
         import subprocess
         import time
         import requests
 
-        subprocess.run(["python3.11", "-m", "pip", "install", "vllm"], check=True)
-
         command = f"nohup python3.11 -m vllm.entrypoints.openai.api_server --model {model_path} &"
-        process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
You can add `start_new_session=True,` to run this in a process group.

Reply:
let's add your changes in a follow-up PR! Let me be done with this one!! 😆